app/views/support/locations/_edit_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= form_with model: [:support, site], local: true, url: form_url, class: "location-form" do |f| %>
```

Comment:
Just realised, how comes we have two location forms? Can't the new and edit share the same one?